simulation/network_simulator.cc

```diff
@@ -1,4 +1,5 @@
 #include "kickcat/OS/Linux/Socket.h"
+#include "kickcat/TapSocket.h"
```

Comment:
Is there updates to put in the readme about how to use the simulator with TapSocket ?

Reply:
Not really, but I'll revert this change specifically (I want to do the switch when the pcap trace will bve available)